### PR TITLE
Update hardware content, add guides menu 

### PR DESCRIPTION
**Pull request by bhawnasehgal**
@lorforlinux I have added guides menu to the header, updated graphics and content for Muscle BioAmp BisCute & Candy. Please review and merge.

### CODE REVIEW

File: guides/usage-guides/skin-preparation/index.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+Skin Preparation Guide
+##########################
+
+Why skin preparation is important?
+***********************************
+
+Proper skin preparation is crucial before recording any biopotential signal be it electrocardiography (ECG), electromyography (EMG), electroencephalography (EEG), or electrooculography (EOG).
+
+- ``Reduction of artifacts`` Removes dead skin cells, oils, & other substances that may introduce noise or artifacts into the recorded signals.
```
Comment (lorforlinux) [inline guides/usage-guides/skin-preparation/index.rst:11]: Dead skin cells, oil, and other substances will keep the skin impedance very high, we should do some background check if this also causes artifacts in the signal.

File: guides/usage-guides/skin-preparation/index.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+- ``Electrode-skin contact`` Ensures optimal contact between the electrodes and the skin surface. Improves the conduction of electrical signals from the body to the recording equipment and minimizes impedance.
```
Comment (lorforlinux) [inline guides/usage-guides/skin-preparation/index.rst:12]: This is good!

File: guides/usage-guides/skin-preparation/index.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+========
+
+Wipe away excess gel with the alcohol swabs or wet wipes.
+
+.. warning:: Close your eyes while using the alcohol swabs for EOG recording else it may cause eye redness & irritation.
+
+Step 5
+========
+
+Now you can either use gel electrodes or the BioAmp bands for the signal recording.
```
Comment (lorforlinux) [inline guides/usage-guides/skin-preparation/index.rst:72]: We should link `BioAmp Bands` to the BioAmp Bands usage guide or buy page.

File: guides/usage-guides/skin-preparation/index.rst [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+- **Using gel electrodes:**
```
Comment (lorforlinux) [inline guides/usage-guides/skin-preparation/index.rst:74]: Should we convert this to a heading?

File: guides/usage-guides/skin-preparation/index.rst [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+  Connect the BioAmp cable to gel electrodes, peel the plastic backing from electrodes and place the IN+, IN-, REF cables according to your specific biopotential recording.
```
Comment (lorforlinux) [inline guides/usage-guides/skin-preparation/index.rst:76]: We should add some graphics here.

File: guides/usage-guides/skin-preparation/index.rst [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+- **Using BioAmp bands:**
```
Comment (lorforlinux) [inline guides/usage-guides/skin-preparation/index.rst:78]: Heading here?

File: hardware/bioamp/bioamp-v1.5/index.rst [hunk 7]
```diff
@@ -169,25 +190,133 @@ You can get your own Bioamp v1.5 bag of parts from `our store <https://store.ups
...
+Step 1: Connecting the cables
+==================================
+
+.. image:: media/board-with-cables.*
+
+Connect the BioAmp cable, 9V snap cable and BioAmp AUX cable to BioAmp v1.5 by inserting the cable ends in the respective JST PH connectors as shown above.
+
+Step 2: Skin Preparation
+==========================
+Apply Nuprep Skin Preparation Gel on the skin surface where electrodes would be placed to remove dead skin cells and clean the skin from dirt. After rubbing the skin surface thoroughly, clean it with a wet wipe.
```
Comment (lorforlinux) [inline hardware/bioamp/bioamp-v1.5/index.rst:209]: Let's keep a blank line after Headings.

File: hardware/bioamp/bioamp-v1.5/index.rst [hunk 8]
```diff
@@ -169,25 +190,133 @@ You can get your own Bioamp v1.5 bag of parts from `our store <https://store.ups
...
+.. image:: media/bioamp-v1-5-emg.*
+
+- **Using Muscle BioAmp Band:**
+
+1. Connect the BioAmp cable to Muscle BioAmp Band in a way such that IN+ and IN- are placed on the arm near the ulnar nerve & REF (reference) on the far side of the band.
+2. Now put a small drop of electrode gel between the skin and metallic part of BioAmp cable to get the best results.
+
+.. note:: These BioAmp Cable connections to gel electrodes/band are specifically for this version of BioAmp v1.5 only. The connections for other BioAmp hardware can be different.
+
+.. note:: In this demonstration we are recording EMG signals from the ulnar nerve, but you can record EMG from other areas as well (biceps, triceps, legs, jaw etc) as per your project requirements. Just make sure to place the IN+, IN- electrodes on the targeted muscle and REF on a bony part.
```
Comment (lorforlinux) [inline hardware/bioamp/bioamp-v1.5/index.rst:233]: Maybe we show this as `tip` instead of `note`?

File: hardware/bioamp/bioamp-v1.5/index.rst [hunk 9]
```diff
@@ -169,25 +190,133 @@ You can get your own Bioamp v1.5 bag of parts from `our store <https://store.ups
...
+
+Step 4: Connecting 9V battery
+===============================
+
+Connect any 9V battery to BioAmp v1.5 using the 9V snap cable. Now activate the board by flipping ON the power switch, and you'll notice an LED light up, showing that the board is ready to use.
```
Comment (lorforlinux) [inline hardware/bioamp/bioamp-v1.5/index.rst:238]: We should mention the type of 9V battery

File: hardware/bioamp/bioamp-v1.5/index.rst [hunk 10]
```diff
@@ -169,25 +190,133 @@ You can get your own Bioamp v1.5 bag of parts from `our store <https://store.ups
...
+2. Flex your muscle to be able to record the muscle signals.
+3. If you want to extract that data then it will be saved by default as a .wav file but you can convert it in any other format according to your project requirements.
+
+.. image:: media/emg-in-mobile-2.*
+
+Using Backyard Brains' Spike Recorder
+--------------------------------------
+
+1. Download the `Spike Recorder App <https://play.google.com/store/apps/details?id=com.backyardbrains&pli=1>`_ from playstore.
+2. Open the app, click the setting icon on the top right corner and set the low band pass filter to 72Hz and high band pass filter to 720Hz.
```
Comment (lorforlinux) [inline hardware/bioamp/bioamp-v1.5/index.rst:285]: iirc, on mobile app we have direct option to set the recording type to EMG/ECG

File: hardware/bioamp/muscle-bioamp-candy/index.rst [hunk 11]
```diff
@@ -75,33 +71,108 @@ Images below shows a quick overview of the hardware design.
...
+****************
 
-.. youtube:: lPX2TGBcHOA
-    :width: 100%
+Step 1: Connect Arduino UNO R3
+=================================
+
+.. image:: media/arduino-candy-connection.*
+
+Connect VCC to either 5V or 3.3V, GND to GND, and OUT to Analog pin A0 of your Arduino UNO via jumper cables provided by us. If you are connecting OUT to any other analog pin, then you will have to change the INPUT PIN in the arduino sketch accordingly.
```
Comment (lorforlinux) [inline hardware/bioamp/muscle-bioamp-candy/index.rst:102]:
```suggestion
Connect `VCC` to either `5V` or `3.3V`, `GND` to `GND`, and `OUT` to `Analog pin A0` of your Arduino UNO via jumper cables provided by us. If you are connecting OUT to any other analog pin, then you will have to change the `INPUT PIN` macro in the example arduino sketch accordingly.
```

File: hardware/bioamp/muscle-bioamp-candy/index.rst [hunk 12]
```diff
@@ -75,33 +71,108 @@ Images below shows a quick overview of the hardware design.
...
+
+**Note:** For demonstration purposes we are showing connections of the sensor with Arduino UNO R3 but you can use any other development board or a standalone ADC of your choice.
```
Comment (lorforlinux) [inline hardware/bioamp/muscle-bioamp-candy/index.rst:104]:
```suggestion
.. note:: For demonstration purposes we are showing connections of the sensor with Arduino UNO R3 but you can use any other development board or a standalone ADC of your choice.
```

File: hardware/bioamp/muscle-bioamp-candy/index.rst [hunk 13]
```diff
@@ -75,33 +71,108 @@ Images below shows a quick overview of the hardware design.
...
+========================================
+
+.. image:: media/candy-cable-connection.*
+
+Connect the BioAmp cable to Muscle BioAmp Candy by inserting the cable end in the JST PH connector as shown above.
+
+Step 4: Skin Preparation
+===============================================
+
+Apply Nuprep Skin Preparation Gel on the skin surface where electrodes would be placed to remove dead skin cells and clean the skin from dirt. After rubbing the skin surface thoroughly, clean it with a wet wipe.
```
Comment (lorforlinux) [inline hardware/bioamp/muscle-bioamp-candy/index.rst:118]: Alcohol wipe is our preference but wet wipe can also be used.

File: hardware/bioamp/muscle-bioamp-candy/index.rst [hunk 14]
```diff
@@ -75,33 +71,108 @@ Images below shows a quick overview of the hardware design.
...
     :align: center
+    :width: 100%
 
-Some project ideas
-********************
+.. note:: In this demonstration we are recording EMG signals from the ulnar nerve, but you can record EMG from other areas as well (biceps, triceps, legs, jaw etc) as per your project requirements. Just make sure to place the IN+, IN- electrodes on the targeted muscle and REF on a bony part.
+
+**Uploading the code:**
+
+Connect your Arduino UNO to your laptop using the USB cable (Type A to Type B). Copy paste any one of the arduino sketches given below in Arduino IDE v1.8.19 that you downloaded earlier:
```
Comment (lorforlinux) [inline hardware/bioamp/muscle-bioamp-candy/index.rst:150]:
```suggestion
Connect your Arduino UNO R3 to your laptop using the USB cable (Type A to Type B). Copy paste any one of the arduino sketches given below in Arduino IDE v1.8.19 that you downloaded earlier:
```

**Comment by lorforlinux** [comment]
@bhawnasehgal Thank you so much for the contribution, I have provided some feedback on improving the text, please check it once and resolve the comments whenever you get a chance.

File: guides/usage-guides/skin-preparation/index.rst (hunk 3 above)
Comment (bhawnasehgal) [reply guides/usage-guides/skin-preparation/index.rst:72]: I won't be able to edit it in this PR but I'm creating an issue for this and will include it in the next PR.

Comment (bhawnasehgal) [reply guides/usage-guides/skin-preparation/index.rst:72]: I have created an issue #36 to track this change

File: guides/usage-guides/skin-preparation/index.rst (hunk 4 above)
Comment (bhawnasehgal) [reply guides/usage-guides/skin-preparation/index.rst]: Yes, that sounds good. Made the commit.

File: guides/usage-guides/skin-preparation/index.rst (hunk 5 above)
Comment (bhawnasehgal) [reply guides/usage-guides/skin-preparation/index.rst:77]: Working on it. I'm creating an issue #37  for the same

File: guides/usage-guides/skin-preparation/index.rst (hunk 6 above)
Comment (bhawnasehgal) [reply guides/usage-guides/skin-preparation/index.rst]: Already made the commit

File: hardware/bioamp/bioamp-v1.5/index.rst (hunk 7 above)
Comment (bhawnasehgal) [reply hardware/bioamp/bioamp-v1.5/index.rst:209]: Sure. Resolving this.

File: hardware/bioamp/bioamp-v1.5/index.rst (hunk 8 above)
Comment (bhawnasehgal) [reply hardware/bioamp/bioamp-v1.5/index.rst]: Sure, resolving this.

File: hardware/bioamp/bioamp-v1.5/index.rst (hunk 10 above)
Comment (bhawnasehgal) [reply hardware/bioamp/bioamp-v1.5/index.rst]: Resolving this.

File: guides/usage-guides/skin-preparation/index.rst (hunk 1 above)
Comment (lorforlinux) [reply guides/usage-guides/skin-preparation/index.rst]: I have fixed this content.

**Comment by bhawnasehgal** [comment]
Thanks for the feedback @lorforlinux, I have resolved most of the suggestions and created issues for the rest.

**Pull request merged**